lxd/storage_pools_config.go

```diff
@@ -103,6 +103,10 @@ func storagePoolFillDefault(name string, driver string, config map[string]string
 	if driver != "dir" {
 		var size uint64
 		if config["size"] == "" {
+			if driver == "lvm" {
+				return nil
+			}
```

Comment:
So if size isn't set and we're on LVM we just skip the rest of the fill function?